blogposts/2021/release-3.24.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Sourcegraph 3.24 release"
+publishDate: 2021-01-20T10:00-07:00
+description: "Sourcegraph 3.24 is released."
+tags: [blog, release]
+slug: "release/3.24"
+published: true
+changelogItems:
+  - description: Added panels to the Sourcegraph monitoring dashboards
+    url: https://docs.sourcegraph.com/admin/observability/metrics#grafana
+    category: Admin
+  - description: Suggested filters in the search results page can now be scrolled.
+    url: https://github.com/sourcegraph/sourcegraph/pull/17097
+    category: Search
+  - description: Dashboard links now included in monitoring alerts.
```

Comment:
```suggestion
  - description: Dashboard links included in monitoring alerts are now more specific.
```

Reply:
The links have existed for a while, the change is to make them more specific and helpful

Reply:
@bobheadxi do you have more any more detail on what makes the dashboard links more specific and helpful?

Reply:
From the changelog:

Dashboard links included in monitoring alerts now:
- link directly to the relevant Grafana panel, instead of just the service dashboard. #17014
- link to a time frame relevant to the alert, instead of just the past few hours. #17034

I think this entry could point out both, e.g. "Links now lead directly to the relevant panel and time frame"

Reply:
```suggestion
  - description: Dashboard links included in monitoring alerts now point directly to the relevant panel and time frame.
```